packages/core/src/lib/wallet-selector.types.ts

```diff
@@ -1,9 +1,9 @@
 import { Observable } from "rxjs";
```

Comment:
I think this should be `import type` too?

Reply:
This is similar to the comment about Subscription:

```ts
export declare class Observable<T> implements Subscribable<T> {}
```